### PR TITLE
[ENH] Adding MLPClassifier

### LINKED ISSUE #3192: [ENH] Adding MLP DL model for classification
**Is your feature request related to a problem? Please describe.**
On the topic of migrating sktime-dl models to sktime, adding MLP model for classification from sktime-dl

### PR DESCRIPTION
**Pull request by AurumnPegasus**
#### Reference Issues/PRs

Fixes #3192 

#### What does this implement/fix? Explain your changes.
Migrating MLPClassifier from sktime-dl to sktime

##### For all contributions
- [x] The PR title starts with either [ENH], [MNT], [DOC], or [BUG] indicating whether the PR topic is related to enhancement, maintenance, documentation, or bug.

##### For new estimators
- [ ] I've added the estimator to the online documentation.
- [ ] I've updated the existing example notebooks or provided a new one to showcase how my estimator works.

### CODE REVIEW

**Comment by AurumnPegasus** [comment]
Reminder: Need to change documentation of `MLPNetwork`

**Comment by fkiraly** [comment]
if you think it´s ready, kindly turn it to "ready to review"

File: sktime/networks/mlp.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+"""Multi Layer Perceptron (MLP) (minus the final output layer)."""
+
+__author__ = ["James-Large, Withington", "AurumnPegasus"]
```
Comment (fkiraly) [inline sktime/networks/mlp.py:4]: these should be three authors, I think

File: sktime/registry/_base_classes.py [hunk 2]
```diff
@@ -81,6 +81,8 @@
     _SeriesToSeriesTransformer,
 )
 
+from ..networks.base import BaseDeepNetwork
```
Comment (fkiraly) [inline sktime/registry/_base_classes.py:84]: please use absolute imports

File: sktime/networks/mlp.py (hunk 1 above)
Comment (AurumnPegasus) [reply sktime/networks/mlp.py:4]: okay, will do

File: sktime/registry/_base_classes.py (hunk 2 above)
Comment (AurumnPegasus) [reply sktime/registry/_base_classes.py:84]: Ah, this seems to be a mistake by vsocde autocomplete. Will resolve this

**Comment by AurumnPegasus** [comment]
Hey @fkiraly ,
The error seems to be a spordiac one as originally reported by you in #2467 
Tagging you since resolving this would let me merge the other pending classifiers as well.

**Review by fkiraly — APPROVED** [review]
Great, thanks!

**Pull request merged**